chemprop/cli/predict.py

```diff
@@ -140,13 +145,9 @@ def add_predict_args(parser: ArgumentParser) -> ArgumentParser:
 
 
 def process_predict_args(args: Namespace) -> Namespace:
-    if args.output is None:
-        args.test_path = Path(args.test_path)
-        name = f"{args.test_path.stem}_preds.csv"
-        args.output = args.test_path.with_name(name)
-    else:
-        args.output = Path(args.output)
-
+    args.output = args.output or Path(args.test_path.stem + "_preds.csv")
```

Comment:
this will write to `$CWD/TEST_PATH_preds.csv`. Is this intended? I'm in favor of writing to the `test_path` directory to keep the predictions with their inputs by default.

Reply:
This was intended. I assumed that if a user has their datasets in a separate folder (e.g. chemprop/tests/data/) and don't specify where to put their predictions, they would rather get the predictions where they are, than have to go find them. But thinking about it more, it is easier to change from the default to cwd than to change from default to data directory, so I'll change this.